apps/amo/models.py

```diff
@@ -368,6 +368,11 @@ def reload(self):
                 pass
         return self
 
+    @classmethod
+    def get_unfiltered_manager(cls):
```

Comment:
Now that there are only 2 choices I find the existence of this method confusing. It's only used in one place, do you really think we need it ?